File: patterns/java/src/refactoring_guru/patterns/builder/example/components/TripComputer.java
package refactoring.guru.patterns.builder.example.components;

import refactoring.guru.patterns.builder.example.cars.Car;

public class TripComputer {

    public void condition() {
        if (Engine.isStarted()) {
            System.out.println("Car is started");
        } else {
            System.out.println("Car isn't started");
        }
    }

    public void fuelLevel() {
        System.out.println("Level of fuel - " + Car.getFuel());
    }
}

File: patterns/cs/src/RefactoringGuru/Builder/Example/Components/TripComputer.cs
using System;
using Builder.Example.Cars;

namespace Builder.Example.Components
{
    /// EN: Just another feature of a car.
	/// 
	/// RU: Одна из фишек автомобиля.
	public class TripComputer
    {
        public virtual void Condition()
        {
            if (Engine.IsStarted())
            {
                Console.WriteLine("Car is started");
            }
            else
            {
                Console.WriteLine("Car isn't started");
            }
        }

        public void FuelLevel()
        {
            Console.WriteLine("Level of fuel - " + Car.GetFuel());
        }
    }
}
